app/services/bot_checker.py

```diff
@@ -23,35 +23,35 @@ def check_account(self, username):
 
         if prediction_db is None:
             _logger.info("This account is not exist in DB")
-            user = self._scraper.get_user_by_username(username)
+            user = self._scraper.get_user_by_username(username)._json
```

Comment:
Why?

Reply:
got problem inside [`test_approve_report()`](https://github.com/dirtyg2120/thesis-backend/blob/8d665c1c389dd1d04bcdc7688f58d770d4411c1a/app/tests/test_report.py#L132).

The old [`user_info()`](https://github.com/dirtyg2120/thesis-backend/blob/8d665c1c389dd1d04bcdc7688f58d770d4411c1a/app/tests/helpers/mock_models.py#L14) return `tweepy.User` which doesn't have full attribute of `user` (`tweepy.models.User` does) 
--> `user._json["status"] = tweets_dict_list` in [`get_full_details()`](https://github.com/dirtyg2120/thesis-backend/blob/8d665c1c389dd1d04bcdc7688f58d770d4411c1a/app/services/scrape.py#L162) run normally in actual app but error on the test case.

Reply:
You can revert back to the old `user_info()` and try to call `user._json` inside a test to see the error.

Reply:
Then fix `MockData`

Reply:
yea i fixed it, our `get_user_by_username()` return `tweepy.models.User`, so it can not call like `user.id_str` anymore, must change to `user._json["id_str"]`. I don't know why it was right before 🤔

Reply:
Because `tweepy.models.User` have attribute `id_str` from JSON response?

Reply:
oh got it, `MockData` is not completely fixed :))))). But the app and test run okay now, do u still want to fix it :)))) @phhai-hcmut

Reply:
also check my mess on slack :V, a small question